docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,120 @@ Followings are the available elements:
+        - A_VALUE: Decided based on a single scalar value.
+        - TENSOR_AVERAGE_VALUE: Decided based on an average value of a specific tensor.
+        - CUSTOM: Decided based on a user-defined callback.
+      - `compared-value-option`: Specifies an element of the nth tensor or you can pick one from the tensors.
+        - [C][W][H][B],n: used for A_VALUE of the compared-value, for example 0:1:2:3,0 means [0][1][2][3] value of first tensor.
+        - nth tensor: used for TENSOR_AVERAGE_VALUE of the compared-value, and specifies which tensor is used.
+      - `supplied-value`: Specifies the supplied value (SV) from the user.
+        - SV
+        - SV1, SV2 (used for RANGE operators)
+      - `operator`: Specifies a comparison operator which is used for comparing the compared-value (CV) and the supplied-value (SV).
+        - EQ: Check if CV == SV
+        - NEQ: Check if CV != SV
+        - GT: Check if CV > SV
+        - GE: Check if CV >= SV
+        - LT: Check if CV < SV
+        - LE: Check if CV <= SV
+        - RANGE_INCLUSIVE: Check if SV1 <= CV and CV <= SV2
+        - RANGE_EXCLUSIVE: Check if SV1 < CV and CV < SV2
+        - NOT_IN_RANGE_INCLUSIVE: Check if CV < SV1 or SV2 < CV
+        - NOT_IN_RANGE_EXCLUSIVE: Check if CV <= SV1 or SV2 <= CV
+      - `then`: Specifies the action, if the result is TRUE.
+        - PASSTHROUGH: Does not let you make changes to the buffers. Buffers are pushed straight through.
+        - SKIP: Does not let you generate the output frame (frame skip).
+        - TENSORPICK: Lets you choose the nth tensor among the input tensors.
+          ```
+          [ tensor 0 ]
+          [ tensor 1 ]  ->   tensor if    ->    [ tensor 0 ]
+          [ tensor 2 ]    (tensorpick 0,2)      [ tensor 2 ]
+          input tensors                         output tensors
+          ```
+      - `then-option`: Specifies the option for TRUE action.
+        - nth tensor: used for TENSORPICK option, for example, `then-option`=0,2 means tensor 0 and tensor 2 are selected as output tensors among the input tensors.
+      - `else`: Specifies the action, if the result is FALSE.
+        - PASSTHROUGH: Does not let you make changes to the buffers. Buffers are pushed straight through.
+        - SKIP: Does not let you generate an output frame (frame skip).
+        - TENSORPICK: Lets you choose the nth tensor among the input tensors.
+      - `else-option`: Specifies the option for FALSE action.
+        - nth tensor: used for TENSORPICK option, for example, `else-option`=0,2 means tensor 0 and tensor 2 are selected as output tensors among the input tensors.
+
+    - **Example launch line with simple if-condition**
+
+      If a specific value ([3][4][2][5] value of the first tensor) is between a given range [10,100], input buffers are pushed straight through. If the value is not in given range, tensor 0 and tensor 2 are selected as output tensors. It can be expressed as:
```

Comment:
```suggestion
     If a specific value, for example, [3][4][2][5] of the first tensor is between a given range of [10,100], input buffers are pushed straight through. If the value is not in the given range, tensor `0` and tensor `2` are selected as output tensors. It can be expressed as:

Reply:
Updated.